packages/client/test/util/parse.spec.ts

```diff
@@ -102,4 +102,16 @@ tape('[Util/Parse]', (t) => {
       'consensus config matches'
     )
   })
+
+  t.test(
+    'should generate expected hash with london block zero and base fee per gas defined',
+    async (t) => {
+      const json = require('../testdata/post-merge.json')
+      const params = await parseCustomParams(json, 'post-merge')
+      t.equals(
+        params.genesis.hash,
+        '0x3b8fb240d288781d4aac94d3fd16809ee413bc99294a085798a589dae51ddd4a'
```

Comment:
Cool! I think for clarity, it would be great if you add an extra check that the base fee of this block is correct as defined in `post-merge.json`. (This is of course part of the hash, and is thus implicitly included in the test, but this would improve clarity)

Reply:
Sure!